### PR TITLE
fix(twitter): misc, support shinigami eyes extension

### PR DESCRIPTION
**Pull request by watatomo**

### CODE REVIEW

**Comment by isabelroses** [comment]
@watatomo is this ready for review?

**Review by uncenter — COMMENTED** [review]
I'd prefer it if we didn't use new `@-moz-document`s for each "section" or feature of a website. Let's try to keep it all in one :)

File: styles/twitter/catppuccin.user.css
Comment (isabelroses) [inline styles/twitter/catppuccin.user.css:1]: This file seems to contain all the metadata and maybe more from the trinket theme and not the twitter theme.

**Comment by uncenter** [comment]
Last valid commit was [`0a60dcd` (#753)](https://github.com/catppuccin/userstyles/pull/753/commits/0a60dcd57928d846e8bbb32b31747d2b2f2fe2f3). Closing this for now.

**Comment by uncenter** [comment]
Updated to that last commit I mentioned so the diff is clearer.

**Pull request closed without merging**